src/switchroot/ostree-prepare-root.c

```diff
@@ -249,7 +246,7 @@ main (int argc, char *argv[])
     }
 
   OstreeComposefsMode composefs_mode = OSTREE_COMPOSEFS_MODE_MAYBE;
-  char *ot_composefs = read_proc_cmdline_key ("ot-composefs");
+  char __attribute__ ((cleanup (free_char))) *ot_composefs = read_proc_cmdline_key ("ot-composefs");
```

Comment:
This is cool, but ideally split up different fixes in separate commits

Reply:
They are separate now